shared/src/packages/sites/libs/enums/sites-api-path.enum.ts

```diff
@@ -1,6 +1,7 @@
 const SitesApiPath = {
   ROOT: '/',
   SECTIONS_BY_SITE: '/:siteId/sections',
+  PROJECT_PROJECT_ID: '/project/:projectId',
```

Comment:
```suggestion
  PROJECT_$PROJECT_ID: '/project/:projectId',
```
all dynamic parameters should start with `$`